code/game/objects/effects/effect_system.dm

```diff
@@ -1039,6 +1047,9 @@ steam.start() -- spawns the effect
 // wall formed by metal foams
 // dense and opaque, but easy to break
 
+var/const/MFOAM_ALUMINUM 	= 1

```

Comment:
I mean we've got global constants elsewhere and these are namespaced, so what's the problem